src/App/Entity/PlacedComponentEx.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace App
+{
+struct PlacedComponentEx : Red::IPlacedComponent
```

Comment:
You don't need extra struct if you just want to register a property. Example: https://github.com/psiberx/cp2077-codeware/blob/main/src/App/Depot/MeshEx.hpp